### ISSUE TITLE
Titles don't work in Scoreboard. Only Chat

### ISSUE THREAD

**Issue by Wh1t3Rose** [fluffyst/Surftimer#114]
Hey I have an issue where Titles do not work in the Scoreboard, Only in Chat, I have there settings set: 

```ck_enforce_default_titles "3```

default_titles:
```
"Default Titles"
{
  "0"
  {
    "steamid" "MYSTEAMID"
    "title" "{default}[{red}Owner{default}]"
    "type" "both"
  }
  "1"
  {
    "flag" "l"
    "title" "{default}[{blue}H-Admin{default}]"
    "type" "both"
  }
  "2"
  {
    "flag" "d"
    "title" "{default}[{gold}Admin{default}]"
    "type" "both"
  }
  "3"
  {
    "flag" "t"
    "title" "{default}[{purple}Mod{default}]"
    "type" "both"
  }
}
```

**Comment by fluffyst** [comment]
ck_enforce_default_titles only accepts 0/1 (false/true)

**Comment by totles** [comment]
we have the same issue. 
set to: `ck_enforce_default_titles "1"`
also if we change the type in `default_titles` to `scoreboard` it also doesnt work

**Comment by Wh1t3Rose** [comment]
> 
> 
> ck_enforce_default_titles only accepts 0/1 (false/true)
> 
> But I guess anything above 1 would be considered true, so it's probably not intended

Then you should update your config file to reflect that as that is not what the config file says

**Comment by totles** [comment]
> Then you should update your config file to reflect that as that is not what the config file says

`// Sets whether default titles will be enforced on clients, Enable / Disable`

no update in the config needed it says everything there is - 0 false/disabled and 1 true/enabled

**Comment by Wh1t3Rose** [comment]
> 
> 
> > Then you should update your config file to reflect that as that is not what the config file says
> 
> `// Sets whether default titles will be enforced on clients, Enable / Disable`
> 
> no update in the config needed it says everything there is - 0 false/disabled and 1 true/enabled

For some reason my says 

` Sets whether default titles will be enforced on clients, 0 to disable, 1 for chat only, 2 for scoreboard only, 3 for both

I also noticed the call for zone admin was wrong in my config as well which is frustrating as I have been trying to give my admins zone perms for 2 weeks. My config had 

` ck_zones_flag
instead of 
`ck_zoner_flag

**Comment by totles** [comment]
did you install the version from this gitrepo? since when you do and start for the first time it creates the file `surftimer.cfg` with the settings mentioned above or did you use a `surftimer.cfg` or plugin version from somewhere else?

`ck_zoner_flag` is also set right in this version (fluffys repo)

**Comment by Wh1t3Rose** [comment]
> 
> 
> did you install the version from this gitrepo? since when you do and start for the first time it creates the file `surftimer.cfg` with the settings mentioned above or did you use a `surftimer.cfg` or plugin version from somewhere else?
> 
> `ck_zoner_flag` is also set right in this version (fluffys repo)

Yep I did and that's the config I am using. I have had my server since the release of this plugin. My guess is that I am still using the cfg file since his first release before he changed those convars and the plugin never updates the cfg file unless you delete it (cfg autogeneration).

**Comment by Wh1t3Rose** [comment]
Any news on this and why tags don't show in Scoreboard/Voice Chat, Only shows in text Chat. @fluffyst

**Comment by Fytcher** [comment]
Same problem here. Using  default conf. No scoreboard titles for me neither.

**Comment by totles** [comment]
ace fixed this in our fork from fluffy if anyone still has problems with that: https://github.com/totles/z4lab-surftimer/commit/e0012df4f25916bc6502d55806e9124f6c35e521
